package.json

```diff
@@ -30,6 +30,7 @@
     "deasync": "^0.1.14",
     "decamelize": "^2.0.0",
```

Comment:
I guess we don't need to install this package for all APIs as well. I think we can just simply make the keys of raw mock data fields to be camelCase and remove this: https://github.com/osu-mist/express-api-skeleton/pull/49/files#diff-2edb83907c552a66fd1c10b1f2535a92R22
What do you think?